environment.yml

```diff
@@ -12,9 +12,9 @@ dependencies:
   - ipykernel=5.3.4
```

Comment:
Ideally, should this file live in the "nbi" subdirectory ?

Reply:
We could do that, but I kept it outside the sub-directory since we also provide that file for customers to create their conda environment setup locally.